Week2/homework/js-exercises/oddNumbers.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+let numberX;
+
+for(numberX = 0; numberX <= 20; numberX++) {
+    if(numberX % 2 === 0){
+    console.log('The number ' + numberX  + ' is even');
+    }
+        else{console.log('The number ' + numberX  + ' is odd');}
+}
```

Comment:
👍 nice!

nit: mind the code formatting, it is much more readable to keep consistent spacing, something like:
```
for (numberX = 0; numberX <= 20; numberX++) {
  if (numberX % 2 === 0) {
    console.log('The number ' + numberX  + ' is even');
  } else {
    console.log('The number ' + numberX  + ' is odd');
  }
} 
```